def/configmap/main.go

```diff
@@ -60,11 +60,7 @@ func (mkv *multiKeyValue) Set(v string) error {
 	if len(p) != 2 {
 		return fmt.Errorf("%s does not match label=value", v)
 	}
-	if mkv == nil {
```

Comment:
The `mkv` pointer is scoped to the method, so this does nothing outside of the method

Reply:
😰